azurelinuxagent/common/dhcp.py

```diff
@@ -257,7 +257,7 @@ def parse_route(response, option, i, length, bytes_recv):  # pylint: disable=W06
 
 
 def parse_ip_addr(response, option, i, length, bytes_recv):
-    if i + 5 < bytes_recv:  # pylint: disable=R1705
+    if i + 5 < bytes_recv:
```

Comment:
Consider removing the `else` below; I personally would find it more readable.

Reply:
same as above